### PR TITLE
Responsive.css File Revision

### PR DESCRIPTION
**Pull request by SychO9**
I tried to break everything down to small commits so you can make sense of it.

After this #5003 gets merged don't merge this until I make sure changes there aren't lost through this PR

### CODE REVIEW

**Comment by XinYenFon** [comment]
I still don't understand the reasoning of moving the higher resolution to up & lower down... Mades the changes unreadable at this stage, either first swap the resolution position then do the change or don't swap the positions. Horrible PR, hopefully, canceled in near future.

**Comment by Gwenwyfar** [comment]
From a quick glance, that's the intention, it is much better to have higher res down to lower res because it removes duplicate code, is easier to follow and edit and doesn't leave holes between the queries. That was what I had already asked on IRC.

However we should be working mostly with max-width only. Min width should be reserved to situations where we actually need it. This way, it all cascades down nicely. This may still be ok for the purpose of this PR though so no need to make changes just yet. I'll review in detail after RC1 is out.

**Comment by XinYenFon** [comment]
I never said what OP doing is wrong, all I`m saying structural changes should be done separately so the actual coding changes can be reviewed far more easily.

I`m still not sure how having high-res up gonna clean up duplicated code, the same logic applies High2Low or Low2High.

The second part, sure if you think its better.

**Comment by Gwenwyfar** [comment]
Css cascades, and so does max-width. It would only make sense to do low res to high res if you were working with min-width.

**Comment by SychO9** [comment]
> Horrible PR, hopefully, canceled in near future.

I don't really understand your point, I get that you disagree with the media queries order but how does 1 commit out of 9 which you consider bad make the whole PR Horrible ?

Also perhaps I'm wrong but code changes are not that hard to review by clicking on a commit to see the changes.

**Comment by XinYenFon** [comment]
I don't disagree with the change -yet I asked I don't understand the reasoning which explained-, I disagree with the way you (front-end developers) doing it.

If you are reviewing the code changes commit by commit. I think we are done here.

**Comment by Gwenwyfar** [comment]
I'm sorry Antes, but we can do without the grumpy comments. Also, this PR was my idea/request, so if you want to complain, that's not even the right person to do so.

**Comment by Sesquipedalian** [comment]
#5003 has been merged now.

**Comment by SychO9** [comment]
fixed merge conflicts

**Comment by live627** [comment]
Can you rebase this branch in order to resolve conflicts @SychO9 ?

**Comment by SychO9** [comment]
there are no conflicts though

**Comment by live627** [comment]
screenshot_2019-02-14 responsive css file revision by sycho9 pull request 5166 simplemachines smf2 1

**Comment by SychO9** [comment]
oh I can't see it on my end. Anyway I've only done this once before so here goes nothing...

**Comment by Gwenwyfar** [comment]
> Can you rebase this branch in order to resolve conflicts @SychO9 ?

Oh, didn't know it could be done that way. Thanks!

File: Themes/default/css/responsive.css [hunk 1]
```diff
@@ -514,9 +535,9 @@
 		font-size: 0.9em;
 		color: #222;
 	}
-	.board_description {
-		display: none;
-	}
+
```
Comment (Gwenwyfar) [inline Themes/default/css/responsive.css]: There's some whitespace left in here.

**Review by Gwenwyfar — APPROVED** [review]

**Comment by Gwenwyfar** [comment]
There's a little more to do before we can close the issue for general problems on mobile and begin opening individual ones, but this is already be a big first step in that direction, thanks SychO :)

**Comment by Sesquipedalian** [comment]
Folks, it is better NOT to rebase pull requests in order to resolve conflicts. Rebasing a local branch that hasn’t been made public as a PR is fine, but rebasing publicly available code is not a good idea. It can really mess stuff up, and if people contributing to SMF start making a habit of it, sooner or later a mess is inevitable.

The better, safer way to resolve conflicts is described in https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/. By far the majority of conflicts that will be encountered in SMF development will be the simple sort described in the first part of that document. I ask everyone to follow that procedure for resolving conflicts from now on.

**Comment by Gwenwyfar** [comment]
For the sake of clarity, what kind of problems can it give? I thought the main problem with rebase was only with people checking the rebased branch out, after they already have done so before the rebase, then it could possibly mess that person's branch. But the SMF repository is not ever doing that.

**Comment by Sesquipedalian** [comment]
I’ve created a topic about resolving conflicts in Git in the Development Contributors board on simplemachines.org. If anyone wants to discuss the matter, please do so there. :)

**Pull request merged**